README.md

```diff
@@ -1,7 +1,14 @@
-oktasdk-python
-=======================
+[<img src="https://aws1.discourse-cdn.com/standard14/uploads/oktadev/original/1X/0c6402653dfb70edc661d4976a43a46f33e5e919.png" align="right" width="256px"/>](https://devforum.okta.com/)
 
-This SDK allows managing an Okta instance via Python.
+# Okta Python Management SDK
+
+## :rotating_light: ANNOUNCEMENT :rotating_light:
+
+The Python SDK for the Okta Management API is currently in development. Please **star** :star: and **watch** :eyes: this repository for updates!
```

Comment:
Change to include version number (i think 1.0) to be something like `Version 1.0.0 of the Python SDK for the Okta Management API is currently in development`.